### PR TITLE
Update RoleCommands.py

### PR DESCRIPTION
**Pull request by modernNeo**
small fix to make output of whois nicer in logs

### CODE REVIEW

File: commands_to_load/RoleCommands.py [hunk 1]
```diff
@@ -97,10 +97,10 @@ def __init__(self, bot):
             return
         for members in membersOfRole:
             name = members.nick or members.name
-            memberString += name + "\n"
+            memberString += name + "\t"
```
Comment (TitanVJ) [inline commands_to_load/RoleCommands.py]: wats the reason for this?

**Comment by modernNeo** [comment]
@TitanVJ what do you mean?

**Comment by TitanVJ** [comment]
image
this is what the output looks like. dont know wat it'll look like with more people but to me it seems weird looking.

**Comment by modernNeo** [comment]
@TitanVJ I guess your version of "wierd looking" is different from mine, if you dont like it, you are free to reject it, I just thought it would look nicer on all front.

**Comment by TitanVJ** [comment]
its fine. ill adjust wats in the log from actual output on my embeds branch. that way it'll work for both purposes.

**Comment by modernNeo** [comment]
i am not sure what you mean so you can close this PR and then do that in your own PR?

**Comment by TitanVJ** [comment]
ok.

**Pull request closed without merging**